woocommerce-gateway-stripe.php

```diff
@@ -54,6 +54,73 @@ function woocommerce_stripe_wc_not_supported() {
 	echo '<div class="error"><p><strong>' . sprintf( esc_html__( 'Stripe requires WooCommerce %1$s or greater to be installed and active. WooCommerce %2$s is no longer supported.', 'woocommerce-gateway-stripe' ), WC_STRIPE_MIN_WC_VER, WC_VERSION ) . '</strong></p></div>';
 }
 
+/**
+ * WooCommerce country not supported notice.
+ *
+ * @since 4.5.1
+ * @return string
+ */
+function woocommerce_stripe_wc_country_not_supported() {
+	echo '<div class="error"><p><strong>' . __( 'Stripe is not supported in your store\'s country.', 'woocommerce-gateway-stripe' ) . '</strong></p></div>';
+}
+
+/**
+ * Check that the WooCommerce country is supported by Stripe.
+ * See https://stripe.com/global for list.
+ *
+ * @since 4.5.1
+ * @return bool
+ */
+function woocommerce_stripe_wc_country_is_supported_country() {
+	$wc_default_country = substr( get_option( 'woocommerce_default_country' ), 0, 2 );
+
+	$supported_countries = apply_filters(
+		'wc_stripe_supported_countries',
+		array(
+			'AT', // Austria
+			'AU', // Australia
+			'BE', // Belgium
+			'BG', // Bulgaria
+			'CA', // Canada
+			'CY', // Cyprus
+			'CZ', // Czech Republic
+			'DK', // Denmark
+			'EE', // Estonia
+			'FI', // Finland
+			'FR', // France
+			'DE', // Germany
```

Comment:
I purposely did NOT put these in alphabetical order by country code, but alphabetical order by country name, to make comparing against Stripe's online list easier in the future.